### PR TITLE
Simplify Linux install overview

### PR DESCRIPTION
**Pull request by richlander**
I had a few goals here.

- Put the focus on getting distro-specific install instructions.
- Elevate the duality of official distro archives and packages.microsoft.com and prioritize them in that order.
- Remove all the support information. It is super confusing. Replace it with a link to our OS support pages on GH.

[internal preview link](https://review.docs.microsoft.com/en-us/dotnet/core/install/linux?branch=pr-en-us-29462)

### CODE REVIEW

File: docs/core/install/linux.md [hunk 1]
```diff
@@ -1,200 +1,68 @@
...
-Ubuntu uses APT (Advanced Package Tool) as a package manager.
+These packages are supported by Microsoft.
 
-The following table is a list of currently supported .NET releases and the versions of Ubuntu they're supported on. These versions remain supported until either the version of [.NET reaches end-of-support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or the version of [Ubuntu reaches end-of-life](https://wiki.ubuntu.com/Releases).
+If .NET is available via both the official package archive for a distribution and packages.microsoft.com, you are encouraged to use the package from the official archive.
 
-- A ✔️ indicates that the version of Ubuntu or .NET is still supported.
-- A ❌ indicates that the version of Ubuntu or .NET isn't supported on that Ubuntu release.
-- When both a version of Ubuntu and a version of .NET have ✔️, that OS and .NET combination is supported.
+## Other Distros
```
Comment (adegeo) [inline docs/core/install/linux.md:60]: I don't think we use the word distro in docs
```suggestion
## Other distributions
```

File: docs/core/install/linux.md [hunk 2]
```diff
@@ -1,200 +1,68 @@
...
 
-| Ubuntu                                   | .NET Core 3.1 | .NET 5   | .NET 6   |
-|------------------------------------------|---------------|----------|----------|
-| ✔️ [22.04](linux-ubuntu.md#2204)       | ❌ 3.1        | ❌ 5.0 | ✔️ 6.0 |
-| ✔️ [21.10](linux-ubuntu.md#2110)       | ✔️ 3.1        | ✔️ 5.0 | ✔️ 6.0 |
-| ✔️ [20.04 (LTS)](linux-ubuntu.md#2004) | ✔️ 3.1        | ✔️ 5.0 | ✔️ 6.0 |
-| ✔️ [18.04 (LTS)](linux-ubuntu.md#1804) | ✔️ 3.1        | ✔️ 5.0 | ✔️ 6.0 |
-| ✔️ [16.04 (LTS)](linux-ubuntu.md#1604) | ✔️ 3.1        | ✔️ 5.0 | ✔️ 6.0 |
+Installation information is als provided for other distributions.
```
Comment (adegeo) [inline docs/core/install/linux.md:62]: bug
```suggestion
Installation information is also provided for other distributions.
```

**Review by adegeo — CHANGES_REQUESTED** [review]
I'm willing to try this since it simplifies the content. However, note that we do have a lot of views on this content (almost 50k a month) and it has good ratings. This would indicate that it is satisfying the needs of the readers.

There are a few things to fix up before merging.

**Comment by richlander** [comment]
I don't doubt that some people like the current format. It has a lot of information and it is (mostly) accurate. I have to believe that others are overwhelmed by all the information. There is a lot more information than is needed for folks to be successful. Last,  all of this support information take a lot of care and feeding. Ideally, we can just rely on it in release notes (which uses a simpler format) and call it good.

Yes, let's see if we can maintain or improve the rating!

**Comment by richlander** [comment]
I believe this topic is ready to go @adegeo.

**Comment by richlander** [comment]
I believe this topic is ready to go @adegeo.

**Review by adegeo — APPROVED** [review]

**Comment by richlander** [comment]
Thanks @adegeo!

**Pull request merged**